atlassian/atlassian-workflow.xml

```diff
@@ -0,0 +1,114 @@
+            <PostEvents path="/getEvents/body/data" source="${/orgID}" />
+        </If>
+
+        <!-- Prepare the lastEventsTime path -->
+        <Set path="/lastEventsTime" value="[]"/>
+
+        <!-- Merge the log data from the API response into the "/lastEventsTime" path. -->
+        <Merge targetPath="/lastEventsTime" sourcePath="/getEvents/body/data/attributes/time" />
+
+        <!-- Set the number of events -->
+        <Set path="/currentFetchEvents" value="${count(/getEvents/body/data/id)}"/>
+        <Set path="/numberOfEvents" value="${count(/getEvents/body/data/id)}"/>
+        <Log type="INFO" message="Page number: ${/pageNumber}, Number of events in this fetched: ${/currentFetchEvents}"/>
+
+        <!-- Prepare the URL within body.links.next for the next requst -->
+        <Set path="/nextLink" value="${/getEvents/body/links/next}" />
+
+        <!-- This loop runs as long as there is a next link available -->
+        <While condition="count(/nextLink) > 0">
+
+            <!-- Make another API call using the next link -->
+            <CallEndpoint url="${/nextLink}" method="GET" savePath="/getEvents">
+                <BearerAuthentication token="${/apiToken}" />
+            </CallEndpoint>
+
+            <!-- Handle Errors -->
+            <If condition="/getEvents/status_code != 200">
+                <Log type="ERROR" message="Atlassian API error: ${/getEvents/body}"/>
+                <Abort reason="Atlassian abort reason: ${/getEvents/body}" />
+            </If>
+
+            <!-- Post Events, if any -->
+            <If condition="count(/getEvents/body/data) > 0">
+                <PostEvents path="/getEvents/body/data" source="${/orgID}" />
+            </If>
+
+            <!-- Set the number of events -->
+            <Set path="/currentFetchEvents" value="${count(/getEvents/body/data/id)}"/>
+            <Set path="/numberOfEvents" value="${/numberOfEvents + /currentFetchEvents}"/>
+
+            <!-- Set the next page number -->
+            <Set path="/pageNumber" value="${/pageNumber + 1}"/>
+            <Log type="INFO" message="Page number: ${/pageNumber}, Number of events in this fetched: ${/currentFetchEvents}"/>
+
+            <!-- Merge the log times from the API response into the "/lastEventsTime" path. -->
+            <Merge targetPath="/lastEventsTime" sourcePath="/getEvents/body/data/attributes/time" />
+
+            <!-- Extract the next link for the next iteration -->
+            <Set path="/nextLink" value="${/getEvents/body/links/next}" />
+        </While>
+
+        <Log type="INFO" message="Total number of fetched events: ${/numberOfEvents}."/>
+
+        <If condition="count(/lastEventsTime) > 0">
+            <!-- Update the bookmark -->
+            <Set path="/lastEvent" value="${max(/lastEventsTime)}" />
+            <ParseDate pattern="yyyy-MM-dd'T'HH:mm:ss[.SSS]'Z'" timeZone="${/timeZone}" date="${max(/lastEventsTime)}" savePath="/lastEventTime" />
```

Comment:
You've set `lastEvent`, but haven't used it and instead ran `max()` twice.

Reply:
fixed